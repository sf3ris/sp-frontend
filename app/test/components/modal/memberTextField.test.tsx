import React from 'react';
import { shallow } from 'enzyme';
import MemberTextField from '../../../src/component/member/Modal/memberTextField.component';

describe('Member Text Field component', () => {

    const onChangeFn = jest.fn();

    it('Should match snapshot', () => {

        const wrapper = shallow(
            <MemberTextField
                value="value"
                onChange={onChangeFn}
                label="label" />
        );

        expect(wrapper.debug()).toMatchSnapshot();

    })
  
})
