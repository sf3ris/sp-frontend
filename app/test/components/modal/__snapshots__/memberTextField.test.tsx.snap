// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Member Text Field component Should match snapshot 1`] = `"<WithStyles(ForwardRef(TextField)) disabled={[undefined]} className=\\"makeStyles-textField-1\\" InputLabelProps={{...}} inputProps={{...}} margin=\\"normal\\" label=\\"label\\" value=\\"value\\" onChange={[Function: onChange]} />"`;
